src/utils.ts

```diff
@@ -58,3 +60,10 @@ export const splitArray = (array: Array<any>, chunks: number): Array<any> =>
 		all[ch] = [].concat(all[ch] || [], one);
 		return all;
 	}, []);
+
+export const reqToPageParameters = (req: Request, filterKeys?: Array<string>): { [key: string]: unknown } => {
```

Comment:
`Array<string>` vs `string[]`

I think we have different styles for the Typescript type, should we unite for 1 style?